server_admin/topics/clients/oidc/con-advanced-settings.adoc

```diff
@@ -88,3 +88,13 @@ The procedure to select the algorithm is:
 . Open the client's *Settings* tab.
```

Comment:
Should be "Click" not "Open"

Reply:
As this is text that was not added or changed by me, I would prefer not to change it as I am not sure if Click or Open is the correct wording.